main-project/src/members/entity/guest-members.entity.ts

```diff
@@ -0,0 +1,24 @@
+import { Meeting } from 'src/meetings/entity/meeting.entity';
+import {
+  BaseEntity,
+  Column,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity('guest_members')
+export class GuestMembers extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  noo: number;
+
+  @Column()
+  user_no: number;
```

Comment:
```suggestion
  userNo: number;
```

컨벤션은 맞춰주는 게 좋아요~
DB 컬럼명과 일치하지 않아서 발생하는 오류면
@Column({ name : 'user_no' })
입력해주면 됩니다. 컬럼 옵션에 다양한 게 있으니 찾아보면 도움될 거예요 ㅎㅎ